apps/coscrad-frontend/src/utils/generic-components/presenters/multilingual-text-presenter.tsx

```diff
@@ -7,20 +30,44 @@ export interface MultilingualTextPresenterProps {
 export const MultilingualTextPresenter = ({
     text,
 }: MultilingualTextPresenterProps): JSX.Element => {
+    const { defaultLanguageCode } = useContext(ConfigurableContentContext);
+
     const { items } = text;
 
-    if (!Array.isArray(items)) {
-        throw new Error(`invalid input to Multlingual text!: ${text}`);
-    }
+    const textItemWithDefaultLanguage =
+        items.find((item) => item.languageCode === defaultLanguageCode) || null;
+
+    const translations = items.filter((items) => items.languageCode !== defaultLanguageCode);
 
     return (
-        <>
-            {/* TODO We need to separate the original in some way */}
-            {items.map(({ languageCode, text, role }) => (
-                <div key={`${languageCode}-${role}`}>
-                    {`{${languageCode}} [${role}]`} {text}
-                </div>
-            ))}
-        </>
+        <Box>
+            <Accordion defaultExpanded variant="outlined">
```

Comment:
I would recommend that the accordion is closed when the page is first loaded, e.g., remove `defaultExpanded`